src/App.tsx

```diff
@@ -15,71 +15,89 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [name, setName] = useState('Nothing');
+  const [clicked, setClicked] = useState(false);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function clear() {
+    setClicked(false);
+    const tbody: any = document.querySelectorAll('tbody');
```

Comment:
you're trying to write the code like in vanilla (pure) JS - use React possibilities: JS variables are available inside your HTML (JSX) and you may write content/classes dynamically